src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
-body {
+$color-block: #eee;
+$start-days: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+$month-lengths: (28, 29, 30, 31);
+$calendar-width: 706;
+
+.page {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
 }
+
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  cursor: pointer;
+  display: flex;
+  flex-flow: row wrap;
+  margin: 81.5px 151px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), margin should not be used to position elements centrally in both axis. Consider using a different approach such as transforming the element on the X and Y axis or using grid/flexbox properties.